### PR TITLE
Deploy infrastructure to blacklodge

### PR DESCRIPTION
**Pull request by liamtoozer**
# Motivation and Context

We need to be able to trigger automated infrastructure deployments in our Blacklodge pipeline using tagged releases.

# What has changed

* Updated the manual release pipeline to allow manual infrastructure deployment
* Added groups to pipeline
* Renamed `Deploy` job
* Updated README

# How to test?

Fly

# Links

https://trello.com/c/KTWj2B55/419-use-concourse-to-deploy-blacklodge-infrastructure-5

### CODE REVIEW

**Comment by liamtoozer** [comment]
[Use concourse to deploy Blacklodge infrastructure (5)](https://trello.com/c/KTWj2B55/419-use-concourse-to-deploy-blacklodge-infrastructure-5)

File: pipelines/manual-release-pipeline.yml [hunk 1]
```diff
@@ -545,11 +553,104 @@ jobs:
...
+    pubsubsvc,
+    print-file-service,
+    exception-manager,
+    toolbox,
+    event-latency-monitor,
+    rabbitmonitor]
+  plan:
+    - get: every-minute
+
+- name: "Run Terraform"
```
Comment (AdamHawtin) [inline pipelines/manual-release-pipeline.yml:638]: This job needs a failure slack message

File: pipelines/manual-release-pipeline.yml [hunk 2]
```diff
@@ -545,11 +553,104 @@ jobs:
...
+    - task: "Run Terraform"
+      file: census-rm-deploy/tasks/terraform-env.yml
+      params:
+        ADMIN_SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+        ENV: ((gcp-environment-name))
+        VAR_FILE: ./tfvars/census-rm-blacklodge.tfvars
+        KUBERNETES_CLUSTER: rm-k8s-cluster
+      input_mapping: {census-rm-terraform: census-rm-terraform-release}
+
+- name: "Run Rabbit Helm"
```
Comment (AdamHawtin) [inline pipelines/manual-release-pipeline.yml:670]: This job needs a failure slack message

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
I think this pipeline needs groups now to easily view the overview/infrastructure/app deployments as they all get mixed together in the UI.

Regarding the failure slack messages, I see they are missing from other pipelines as well so I'll raise a small ticket to add alerts into other pipelines where they are missing as well.

**Comment by liamtoozer** [comment]
> I think this pipeline needs groups now to easily view the overview/infrastructure/app deployments as they all get mixed together in the UI.

We thought about doing this but wasn't sure if it should have been a separate piece of work as it adds a little more complexity to the ticket (not much though!). Happy to do it though as it shouldn't take long?

**Comment by AdamHawtin** [comment]
@liamtoozer I think we need to keep the pipelines organised, they're already getting pretty big and anything we can do to keep them clear and understandable in the UI helps.

**Review by AdamHawtin — APPROVED** [review]

**Review by hbrace — APPROVED** [review]

**Pull request merged**